### PR TITLE
Icemoon lavaloops now look different.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Lavaland lavaloops and icemoon variant look exactly the same, with the only difference being the boomerang effects, yet they have two different entries in the index of catchable fishes, which has been noticed by a few other players.

This PR differentiates the two, giving the latter new name, sprite, temperature requirements, including two evolution datums for lavaloops to plasmaloops and viceversa.

This also fixes a couple logic errors with the evolution requirements tooltip and the moonfish evolution size/weight reqs.

## Why It's Good For The Game
Having two exact looking lavaloop entries in the fish index is kinda confusing.

## Changelog

:cl:
add: Icemoon lavaloops (now named plasmaloops) fish now look different.
/:cl:

### CODE REVIEW

**Review by Wallemations — APPROVED** [review]
Looks good but it failed a check, feel free to merge it once you fix that

**Comment by Ghommie** [comment]
@Wallemations all check pass now.

**Comment by Wallemations** [comment]
> @Wallemations all check pass now.

image

?

File: code/modules/fishing/fish/fish_evolution.dm [hunk 1]
```diff
@@ -69,8 +69,15 @@ GLOBAL_LIST_EMPTY(fishes_by_fish_evolution)
 /datum/fish_evolution/proc/get_evolution_tooltip()
 	. = ""
 	if(required_temperature_min > 0 || required_temperature_max < INFINITY)
-		var/max_temp = required_temperature_max < INFINITY ? " to [required_temperature_max]" : ""
-		. = "An aquarium temperature of [required_temperature_min][max_temp] is required."
+		var/temp_reqs = ""
+		if(required_temperature_min == 0)
+			temp_reqs = "below [required_temperature_max]"
+		else if(required_temperature_max < INFINITY)
```
Comment (SmArtKar) [inline code/modules/fishing/fish/fish_evolution.dm:75]:
```suggestion
		else if(required_temperature_max == INFINITY)
```

**Review by SmArtKar — CHANGES_REQUESTED** [review]

**Pull request merged**